examples/example-core/pom.xml

```diff
@@ -36,4 +36,37 @@
         <module>example-spring-mybatis</module>
         <module>example-spring-jpa</module>
     </modules>
+
+    <build>
+        <pluginManagement>
+		    <plugins>
```

Comment:
The indents here and below mixed `tab` and `space` and they are still incorrect. How about just removing the `lifecycle-mapping`?

Reply:
Yea, there is no specific use of it(a comment also mentions), I'll remove the `lifecycleMapping`. Also @TeslaCN there are too many changed files in this PR can I do it another PR?

Reply:
I prefer doing it in this PR. The examples updated in https://github.com/apache/shardingsphere/pull/13931.
You may need to update your branch.